crates/chronicle/src/codegen/mod.rs

```diff
@@ -63,12 +70,26 @@ fn gen_type_enums(domain: &ChronicleDomainDef) -> rust::Tokens {
     let graphql_enum = &rust::import("chronicle::async_graphql", "Enum");
     let domain_type_id = &rust::import("chronicle::common::prov", "DomaintypeId");
     let prov_role = &rust::import("chronicle::common::prov", "Role").qualified();
-    quote! {
 
```

Comment:
It would perhaps be tidier to have `domain_docs/` inside `docs/`, but currently we `.dockerignore` the latter.

Reply:
I think it's reasonable to keep them separate here